### ISSUE TITLE
Smart contract verification phase 2 does not accept metadata file generated by Foundry-rs

### ISSUE THREAD

**Issue by quiet-node** [hashgraph/hedera-mirror-node-explorer#756]
### Description

Seems like metadata generated by foundry-rs by is structured differently than other tools like Solc, Hardhat, etc. so the FileImporter doesn't accept the metadata from foundry. 

Here is an example of metadata file generated by foundry
[Counter.json](https://github.com/hashgraph/hedera-mirror-node-explorer/files/13370776/Counter.json)

The contract: 
``` solidity
// SPDX-License-Identifier: UNLICENSED
pragma solidity ^0.8.13;

contract Counter {
    uint256 public number;

    function setNumber(uint256 newNumber) public {
        number = newNumber;
    }

    function increment() public {
        number++;
    }
}
```

And here is the link to the hashscan page of the contract:  https://hashscan-latest.hedera-devops.com/testnet/contract/0.0.5893811

### Steps to reproduce

1. Deploy a contract by foundry-rs
2. Go to the hashscan page of the contract -> click verify contract -> drop the metadata file generated by foundry to the file. chooser -> nothing is showing

### Hedera network

testnet

### Version

staging server

### Operating system

macOS

**Comment by ericleponner** [comment]
Could you add `Counter.sol` to the issue ?

**Comment by quiet-node** [comment]
@ericleponner I have updated the issue to include the contract. The issue does not accept .sol file so I just pasted the raw code there.

**Comment by svienot** [comment]
Will be fixed once #799 is integrated.
Fix available on staging server.

**Comment by ericleponner** [comment]
This issue is fixed by #811.
